app/models/claim/pii_scrubber.rb

```diff
@@ -23,7 +23,11 @@ class PiiScrubber
       :building_society_roll_number,
     ]
 
-    MINIMUM_DATE_OF_LAST_ACTION = 2.months.ago
+    TIME_BEFORE_CLAIM_CONSIDERED_OLD = 2.months
+
+    def initialize
+      @minimum_time_of_last_action = TIME_BEFORE_CLAIM_CONSIDERED_OLD.ago
```

Comment:
I reckon we can forgo this instance variable and just call `TIME_BEFORE_CLAIM_CONSIDERED_OLD.ago` in the scope method below.